actions.yaml

```diff
@@ -1,31 +1,5 @@
 restart:
   description: Restart the Kubernetes master services on demand.
-create-rbd-pv:
```

Comment:
```
$ git grep create-rbd-pv
docs/index.md:      create-rbd-pv
templates/rbd-persistent-volume.yaml:# `create-rbd-pv` action. This is a temporary file on disk to enlist resources.
```

Looks like there's a doc reference to create-rbd-pv that needs to be removed.

It also looks like `templates/rbd-persistent-volume.yaml` was only used by the create-rbd-pv action and can be removed. Yay! Mind doing that?